plugin-session-replay-android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.amplitude.android.plugin.sessionreplay">
```

Comment:
This doesn't really match the module name but I think that `com.amplitude.android.plugin.sessionreplay` is more idiomatic than alternatives.